spring-cloud-starter-stream-source-sftp/src/main/java/org/springframework/cloud/stream/app/sftp/source/SftpSourceProperties.java

```diff
@@ -18,10 +18,11 @@
 import java.io.File;
 import java.util.regex.Pattern;
 
+import javax.validation.constraints.AssertFalse;
 import javax.validation.constraints.AssertTrue;
-import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
 
+import org.hibernate.validator.constraints.NotBlank;
```

Comment:
Hibernates NotBlank is deprecated.  Use  javax.validation.constraints.NotBlank instead.